cmd/minikube/cmd/config/get_test.go

```diff
@@ -21,15 +21,17 @@ import (
 )
 
 func TestGetNotFound(t *testing.T) {
-	createTestConfig(t)
+	createTestConfig(t, "create")
 	_, err := Get("nonexistent")
 	if err == nil || err.Error() != "specified key could not be found in config" {
 		t.Fatalf("Get did not return error for unknown property")
 	}
+	// remove all temp dir
+	createTestConfig(t, "remove")
```

Comment:
I like this PR - but have some suggestions to make this more idiomatic Go:

* have `createTestConfig` return either the path generated, or a cleanup function
* immediately after `createTestConfig` is called, call: `defer removeTestConfig(path)`

This way cleanup is guaranteed, and we omit the stuttering function signature of `create..*create`

Reply:
@tstromberg That's a pretty good idea, I implemented it.